compiler/src/iree/compiler/Codegen/Common/IREEComprehensiveBufferizePass.cpp

```diff
@@ -276,11 +308,7 @@ void addIREEComprehensiveBufferizePasses(
 }
 
 void addConstantBufferizePasses(OpPassManager &funcPassManager) {
-  OneShotBufferizationOptions options;
-  options.copyBeforeWrite = true;
-  options.enforceAliasingInvariants = false;
-  options.opFilter.allowOperation(arith::ConstantOp::getOperationName());
-  funcPassManager.addPass(bufferization::createOneShotBufferizePass(options));
+  funcPassManager.addPass(createIREEBufferizeConstantsPass());
 }
```

Comment:
Do we replace all the addConstantBufferizePasses uses in IREE with `funcPassManager.addPass(createIREEBufferizeConstantsPass());` directly?

Reply:
I was mostly being lazy, done!